hash/msx2_flop.xml

```diff
@@ -7137,6 +7137,18 @@ The following floppies came with the machines.
 		</part>
 	</software>
 
+	<software name="hbiv1">
+		<description>HBI-V1 Video Digitizer (Japan)</description>
+		<year>1989</year>
+		<publisher>Sony</publisher>
+		<info name="usage" value="Requires a Japanese system."/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="hbi-v1 video digitizer - sony.dsk" size="737280" crc="c039a66c" sha1="d720fa4142cac3a34d20dcd90c0273f751213dfa"/>
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Is this actually working?  Doesn’t it require [a frame grabber cartridge](https://www.msx.org/wiki/Sony_HBI-V1) to actually do anything useful?

Reply:
No, it isn't. I have updated the supported state.